packages/react/src/views/EmbeddedChat.js

```diff
@@ -178,6 +222,8 @@ const EmbeddedChat = ({
     [RCInstance, ECOptions]
   );
 
+  if (!isSynced) return null;
```

Comment:
i think we only need this if remoteOpt == true right?

Reply:
i think this will always be false and lead to null if we have remoteOpt false?

Reply:
No, it won't. If `remoteOpt` is false in `useEffect`, it will return early. However, the finally block will still run and change `isSynced` to true.

Reply:
IMO its bit unreadable, lets just refactor it out such that the function only gets called when/if remoteOpt is true else I don't think we need to be calling this function